benchmarks/lockhammer/src/lockhammer.c

```diff
@@ -38,73 +38,96 @@
 #include <pthread.h>
 #include <sys/types.h>
 #include <time.h>
+#include <string.h>
 #include <fcntl.h>
 
+#include "lockhammer.h"
+
 #include ATOMIC_TEST
 
 uint64_t test_lock = 0;
 uint64_t sync_lock = 0;
 uint64_t ready_lock = 0;
 
-struct arg {
-    unsigned long ncores;
-    unsigned long nthrds;
-    unsigned long iter;
-    unsigned long *lock;
-    unsigned long *rst;
-    unsigned long *nsec;
-    unsigned long *depth;
-    unsigned long *nstart;
-    unsigned long hold, post;
-};
-typedef struct arg arg;
-
 void* hmr(void *);
 
+void print_usage (char *invoc) {
+    fprintf(stderr,
+"Usage: %s\n\t[-t threads]\n\t[-a acquires per thread]\n\t" \
```

Comment:
This can be indented as normal, and the trailing \ is not needed for strings to be continued.